en/django_start_project/README.md

```diff
@@ -118,6 +118,35 @@ ALLOWED_HOSTS = ['127.0.0.1', '.pythonanywhere.com']
 
 > Also add `.amazonaws.com` to the `ALLOWED_HOSTS` if you are using cloud9
 
+> If you are hosting your project on `Glitch.com`, let us protect the Django secret key that needs to 
+> remain confidential (otherwise, anyone remixing your project could see it) :
```

Comment:
Putting a space before a colon (`:`) is AFAIK a French convention not usually followed in English.
```suggestion
> remain confidential (otherwise, anyone remixing your project could see it):
```